vendor/a24testLib/testCaseHelpersExtend.js

```diff
@@ -73,6 +73,57 @@ if (typeof a24SetInputDropdownValue === "undefined") {
     }
 }
 
+if (typeof a24SetInputDateValue === "undefined") {
+    var a24SetInputDateValue = function (sFieldName, sValue, iIndex) {
+       a24DoTask(
+           100,
+           function() {
+               if (typeof iIndex === "undefined") {
+                   iIndex = 0;
+               }
+               a24GetElementFromText(
+                   $, "label", sFieldName, true
+               ).eq(iIndex).parent().find(".datepicker").eq(0).trigger("onSetDateForTest", sValue);
+           }
+       );
+    }
+}
+
+if (typeof a24SetInputTelNumberCountryCodeValue === "undefined") {
+    var a24SetInputTelNumberCountryCodeValue = function (sFieldName, sValue, iIndex) {
+        a24DoTask(
+            100,
+            function() {
+                if (typeof iIndex === "undefined") {
+                    iIndex = 0;
+                }
+                a24GetElementFromText(
+                    $, "label", sFieldName, true
```

Comment:
this one will not work since this does not have a label, think `.a24TelInput .a24InputHeader` should work